default.nix

```diff
@@ -75,7 +75,7 @@ stdenv.mkDerivation rec {
     ++ [ ocamlPackages.ounit rsync which ]
   )
   ++ optionals shell (
-    [ jq curl git gnupg ] # Dependencies of the merging script
+    [ jq curl (if shell then gitFull else git) gnupg ] # Dependencies of the merging script
```

Comment:
The `shell` variable is always true here.

See the definition of the [optionals](https://github.com/NixOS/nixpkgs/blob/f38e8c0e96a75862daccc7f978496631dfcf6037/lib/lists.nix#L209) function.

Reply:
Yeah, you can just replace `git` with `gitFull`.

Reply:
oh ok cool